graph/tree_diameter/info.toml

```
@@ -0,0 +1,33 @@
+title = 'Tree Diameter'
+timelimit = 2.0
```

```suggestion
timelimit = 5.0
```

A + Bだけ2sなんですが、このジャッジの問題は基本5sにしています　これ意味不明なんでなんとかしたいですね…

---

5s に変更しました